### PR TITLE
add non-360° video example (fixes #472)

### PR DESCRIPTION
**Pull request by cvan**

### CODE REVIEW

File: examples/video/index.html [hunk 1]
```diff
@@ -8,17 +8,44 @@
...
+             width="160" height="90" autoplay loop="true"
+             crossOrigin="anonymous"></video>
+      <video id="videoHypercube"
+             src="https://ucarecdn.com/210dd043-dcd3-446d-9b36-ad19471f6273/"
+             width="160" height="90" autoplay loop="true"
              crossOrigin="anonymous"></video>
     </a-assets>
 
-    <a-scene stats="true">
+    <a-scene>
```
Comment (dmarcos) [inline examples/video/index.html]: We're missing stats on this one

Comment (ngokevin) [inline examples/video/index.html]: Maybe stats should only be on aframe-core examples?

Comment (cvan) [inline examples/video/index.html]: oh sorry!

Comment (dmarcos) [inline examples/video/index.html]: can you add it? :)

**Comment by cvan** [comment]
rr? @dmarcos

**Pull request merged**